test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -70,12 +71,16 @@ const (
 )
 
 var (
+	extentionPoints = labelValues{
+		label:  extentionPointsLabelName,
+		values: []string{"Filter", "Score"},
+	}
+
 	defaultMetricsCollectorConfig = metricsCollectorConfig{
-		Metrics: []string{
-			"scheduler_scheduling_algorithm_predicate_evaluation_seconds",
-			"scheduler_scheduling_algorithm_priority_evaluation_seconds",
-			"scheduler_e2e_scheduling_duration_seconds",
-			"scheduler_pod_scheduling_duration_seconds",
+		Metrics: map[string]*labelValues{
+			"scheduler_framework_extension_point_duration_seconds": &extentionPoints,
```

Comment:
nit: put the definition of `extentionPoints` inline here.

s/extention/extension

Reply:
Done